src/en/clean-copy/04-Part III. The New Hope/27.md

```diff
@@ -9,48 +9,48 @@ The first half of the 20th century, with all the wars, depressions, and prohibit
 
-One of the moving forces of the lagerization and the consolidation was the incessant process of mergers and acquisitions that had finally culminated in creating the global beer conglomerate AB InBev, which consumed American Annheuser-Busch, Brazilian Ambev (which was in turn the joint venture of Brazilian brewers Antarctica and Brahma), and Belgian Interbrew (which was the result of the Artois-Pidboef merging), joined by SABMiller (which in turn was created after the SAB and Miller deal) in 2016. In some countries, the AB InBev market share exceeds 90% (for example, in Brazil and Uruguay) and even 99% (in the Dominican Republic)[ref](https://www.nbwa.org/resources/look-inside-global-beer-market).
+Unalterable economic forces led to this lagerization and brewery consolidation. Within the new Global markets, the incessant process of mergers and acquisitions had finally culminated in creating the global beer conglomerate AB InBev, which consumed American Annheuser-Busch, Brazilian Ambev (which was in turn the joint venture of Brazilian brewers Antarctica and Brahma), and Belgian Interbrew (which was the result of the Artois-Pidboef merger), joined by SABMiller (which in turn was created after the SAB and Miller deal) in 2016. In some countries, the AB InBev market share exceeds 90% (for example, in Brazil and Uruguay) and even 99% (in the Dominican Republic)[ref](https://www.nbwa.org/resources/look-inside-global-beer-market).
 
-This company — as well as the smaller conglomerates: Heineken N.V., Carlsberg Group, MillerCoors, and China Resources Snow Breweries Ltd. — owns literally hundreds of beer brands. Does it worth mentioning that almost all of these brands are totally identical pale lagers that the consumers aren't able to distinguish one from another[ref:howard-globalization:2]()? In 2012, the four largest brewing companies in the world controlled 70% of the income in the industry[ref:howard-globalization:1]() — the unprecedented digits that surpassed even those from the London porter oligopoly times.
+This company — as well as the smaller conglomerates: Heineken N.V., Carlsberg Group, MillerCoors, and China Resources Snow Breweries Ltd. — owns literally hundreds of beer brands. Does it even bear mentioning that almost all of these brands are totally identical pale lagers that the average consumers aren't able to *truly* distinguish from one another[ref:howard-globalization:2]()? In 2012, the four largest brewing companies in the world controlled 70% of the market revenue[ref:howard-globalization:1]() — the unprecedented centraliztion and market growth had surpassed even those from the London porter oligopoly times.
```

Comment:
income, not revenue ) that's different thing.
Also, 'market growth' is not needed there, I actually don't know those numbers

Reply:
revenue is the same as income, your choice.